recipes/bigdft/build.sh

```diff
@@ -0,0 +1,18 @@
+# Files
+cp $RECIPE_DIR/conda.rc .
+cp $RECIPE_DIR/patches/* .
+
+# Patches
+patch futile/flib/utils.c utils.patch
+patch bigdft/src/output.f90 output.patch
```

Comment:
You can patch the sources directly in the `meta.yaml` file.

Reply:
Thanks, I've made this modification!